PS8lite.Rmd

```diff
@@ -92,8 +93,12 @@ training <- training %>%
 Questions to ask yourself or discuss with your peers
 
 1. Why didn't I apply the same change to the `test` set?
-1. If I transformed the outcome variable now, what will I have to make sure to do later on?
 
+We do not apply the same change to the 'test' set because ultimately, we would like to predict `SalePrice` using the test data set, not `log(SalePrice+1)`. Therefore, there is no need to transform the outcome variable `SalePrice`.
+
+2. If I transformed the outcome variable now, what will I have to make sure to do later on?
+
+If the outcome variable of `test` set is transformed now, we will have to make sure to convert the predicted `log(SalePrice+1)` back to `SalePrice`.
```

Comment:
Hi, everything looks good. Here is an example comment that I would leave as part of a “code review” if this were real life. Then you would make the requested changes to this PR and then I would merge your changes into my master copy of the repo. I won't be doing that last step in this case.